pages/linux/xbps.md

```diff
@@ -0,0 +1,23 @@
+# xbps
+
+> The X Binary Package System (or xbps) is the binary package system used by Void Linux.
+
+- Install packages; Option -S --synchronize:
+
+`xbps-install --synchronize {{package_name1}} {{package_name2}}`
```

Comment:
Since -S is used in different cases for different stuff I guess --synchronize is more accurate and less confusing